samples/boards/arc_secure_services/sample.yaml

```diff
@@ -3,6 +3,16 @@ sample:
      Designware ARC SecureShiled.
   name: Designware ARC Secure monitor
 tests:
-  test:
+  sample.arc_secure_services:
     platform_whitelist: nsim_sem em_starterkit_em7d_secure
-    tags: secure
\ No newline at end of file
+    tags: secure
+    harness: console
+    harness_config:
+      type: multi_line
+      ordered: true
```

Comment:
Don't think the output is deterministic enough for this ordered multi-line list, the number of main thread iterations still finished while switching to the Normal application may vary.
How about a single line filter for "Go to normal application" only?